recipes/cccl/all/conanfile.py

```diff
@@ -18,50 +22,52 @@ class CcclConan(ConanFile):
         "muffle": True,
         "verbose": False,
     }
-    settings = "compiler",
 
-    _source_subfolder = "source_subfolder"
+    @property
+    def _cccl_dir(self):
+        return os.path.join(self.package_folder, "bin")
 
-    def configure(self):
-        if self.settings.compiler != "Visual Studio":
-            raise ConanInvalidConfiguration("This recipe support only Visual Studio")
-        del self.settings.compiler
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def package_id(self):
-        del self.info.options.muffle
-        del self.info.options.verbose
+        self.info.clear()
+
+    def validate(self):
+        if not is_msvc(self):
+            raise ConanInvalidConfiguration("This recipe only supports msvc/Visual Studio.")
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
-                  strip_root=True, destination=self._source_subfolder)
```

Comment:
keep these lines here, sources should be downloaded in the source method (remove them from build method as well)

Reply:
Oops, should have not moved that from source when I moved the patching stuff I think. Moved it back but I'll wait a bit and see if we get a release tag for 1.3 first.